templates/test/ci/cluster-template-prow-external-cloud-provider-ci-version.yaml

```diff
@@ -2,8 +2,7 @@ apiVersion: cluster.x-k8s.io/v1beta1
 kind: Cluster
 metadata:
   labels:
-    ccm: external
-    cni: calico
+    cni: ${CLUSTER_NAME}-calico
```

Comment:
this is what fixes #2341

Reply:
Usage was removed in https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2209

> So the path above was overriding these values?

yes, the external cloud provider patch was modifying the labels and overwriting the cni one.